app/build.gradle

```diff
@@ -61,7 +61,7 @@ dependencies {
     implementation presentationDependencies.glide
     kapt presentationDependencies.glideCompiler
 
-    implementation presentationDependencies.butterKnife
+    implementation presentationDependencies.butterknife
```

Comment:
using Kotlin Synthetic Binding instead of butter knife

Reply:
https://medium.com/coding-blocks/goodbye-findviewbyid-say-hello-to-synthetic-binding-in-kotlin-d3727bd8438e